tests/test_response_service.py

```
@@ -0,0 +1,12 @@
+"""Tests for core.response_service module."""
+
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)